gooddata-java-model/src/main/java/com/gooddata/sdk/model/settings/SettingItem.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2007-2021, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.sdk.model.settings;
+
+import com.fasterxml.jackson.annotation.*;
+import com.gooddata.sdk.common.util.GoodDataToStringBuilder;
+
+import static com.gooddata.sdk.common.util.Validate.notEmpty;
+import static com.gooddata.sdk.common.util.Validate.notNullState;
+
+@JsonTypeName("settingItem")
+@JsonTypeInfo(include = JsonTypeInfo.As.WRAPPER_OBJECT, use = JsonTypeInfo.Id.NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class SettingItem {
+
+    private final String name;
+    private final String source;
+    private String enabled;
```

Comment:
Well this doesnt't have to be only `boolean enabled`, some settings contain various strings ... I would rename it to `value`, create general `get/setValue` + add convenience `boolean isEnabled; setEnabled(boolean)` for cases when it's boolean ;)

Reply:
Fixed and added convenience methods.